### PR TITLE
network: tc: use request queue to configure QDisc or TClass

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: src/libsystemd/sd-netlink/netlink-message-rtnl.c [hunk 1]
```diff
@@ -1072,6 +1072,42 @@ int sd_rtnl_message_new_traffic_control(
         return 0;
 }
 
+int sd_rtnl_message_traffic_control_get_ifindex(sd_netlink_message *m, int *ret) {
+        struct tcmsg *tcm;
+
+        assert_return(rtnl_message_type_is_traffic_control(m->hdr->nlmsg_type), -EINVAL);
```
Comment (bluca) [inline src/libsystemd/sd-netlink/netlink-message-rtnl.c:1080]: assert/check on `m`, `m->hdr` and tcm? Same below

**Review by bluca — APPROVED** [review]
LGTM, just one nit

**Comment by bluca** [comment]
Also would be good to have some tests for coverage, but not sure how feasible it is

**Comment by yuwata** [comment]
@bluca Thank you for the review. The above point is addressed. Upgrading the green label.

> Also would be good to have some tests for coverage, but not sure how feasible it is

We already have tests for traffic control, and now the sections about traffic control are stored in hashmap (hence the order is randomized). If the existing tests are green, then it means the resolving dependencies between sections works fine.

**Pull request merged**